src/containers/search/SearchParticipants.js

```diff
@@ -0,0 +1,171 @@
+// @flow
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+import { List, Map } from 'immutable';
+import {
+  Button,
+  Card,
+  CardHeader,
+  CardSegment,
+  DatePicker,
+  Input,
+  Label,
+  PaginationToolbar,
+  SearchResults,
+  Typography,
+} from 'lattice-ui-kit';
+import {
+  DataUtils,
+  ReduxUtils,
+  useRequestState,
+} from 'lattice-utils';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { SEARCH_PARTICIPANTS, clearSearchedDataAndRequestState, searchParticipants } from './actions';
+
+import { PROPERTY_TYPE_FQNS } from '../../core/edm/constants/FullyQualifiedNames';
+import { PARTICIPANT_PROFILE } from '../../core/router/Routes';
+import { goToRoute } from '../../core/router/RoutingActions';
+import { getPersonFullName } from '../../utils/PeopleUtils';
+import { SEARCH } from '../../utils/constants/ReduxStateConsts';
+
+const { DOB } = PROPERTY_TYPE_FQNS;
+const { SEARCHED_PARTICIPANTS, TOTAL_HITS } = SEARCH;
+const { isFailure, isPending, isSuccess } = ReduxUtils;
+const { getEntityKeyId, getPropertyValue } = DataUtils;
+
+const MAX_HITS = 10;
+const resultLabels = Map({
+  name: 'Name',
+  personDOB: 'Date of Birth',
+});
+
+const SearchGrid = styled.div`
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(4, 1fr);
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 100%;
+
+  span:last-child {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+`;
+
+const SearchParticipants = () => {
+
+  const [formInputs, setFormInputs] = useState({ firstName: '', lastName: '' });
+  const [dob, setDOB] = useState('');
+  const [page, setPage] = useState(1);
+
+  const onInputChange = (e :SyntheticEvent<HTMLInputElement>) => {
+    const { name, value } = e.currentTarget;
+    setFormInputs({ ...formInputs, [name]: value });
+  };
+
+  const dispatch = useDispatch();
+
+  const searchCWPParticipants = (e :SyntheticEvent<HTMLInputElement> | void, startIndex :?number) => {
+    const start = startIndex || 0;
+    dispatch(searchParticipants({
+      dob,
+      firstName: formInputs.firstName,
+      lastName: formInputs.lastName,
+      maxHits: MAX_HITS,
+      start,
+    }));
+  };
+
+  const onPageChange = ({ page: newPage, start } :Object) => {
+    searchCWPParticipants(undefined, start);
+    setPage(newPage);
+  };
```

Comment:
if you remove event parameter from `searchCWPParticipants` function then you don't need to pass `undefined` here.

Reply:
same comment as above ^^